habitat/datasets/pointnav/pointnav_dataset.py

```diff
@@ -96,15 +97,28 @@ def __init__(self, config: Config = None) -> None:
                 data_path=dataset_dir, scene=scene
             )
             with gzip.open(scene_filename, "rt") as f:
-                self.from_json(f.read())
+                # import pdb; pdb.set_trace()
+                self.from_json(f.read(), scenes_dir=config.SCENES_DIR)
 
-    def from_json(self, json_str: str) -> None:
+    def from_json(
+        self, json_str: str, scenes_dir: Optional[str] = None
+    ) -> None:
         deserialized = json.loads(json_str)
         if CONTENT_SCENES_PATH_FIELD in deserialized:
             self.content_scenes_path = deserialized[CONTENT_SCENES_PATH_FIELD]
 
         for episode in deserialized["episodes"]:
             episode = NavigationEpisode(**episode)
+            # import pdb; pdb.set_trace()
+
+            if scenes_dir is not None and episode.scene_id.startswith(
+                DEFAULT_SCENE_PATH_PREFIX
+            ):
+                episode.scene_id = episode.scene_id[
+                    len(DEFAULT_SCENE_PATH_PREFIX) :
+                ]
+                episode.scene_id = os.path.join(scenes_dir, episode.scene_id)
```

Comment:
Should this join be outside the if?  That'll give us flexibility in the future to not have all scene paths start with `data/`